README.md

```
@@ -1,10 +1,23 @@
-# Sargam Icons
-A collection of 1200+ handcrafted open-source icons.
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)